src/main/java/calculator/Coordinator.java

```diff
@@ -0,0 +1,39 @@
+package calculator;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class Coordinator {
+    private final IOService IOService;
+    private final DelimiterService delimiterService;
+    private final NumberService numberService;
+
+    public Coordinator(IOService IOService, DelimiterService delimiterService, NumberService numberService) {
+        this.IOService = IOService;
+        this.delimiterService = delimiterService;
+        this.numberService = numberService;
+    }
+
+    public void execute() {
+        try {
+            // 1. 입력 받기
+            String input = IOService.getInput();
+
+            // 2. 구분자 처리 및 숫자 추출
+            List<String> numberStrings = delimiterService.processInputAndExtractNumbers(input);
+
+            // 3. 숫자 처리 및 계산
+            List<BigDecimal> numbers = numberService.processNumbers(numberStrings);
+            BigDecimal result = numberService.calculateSum(numbers);
```

Comment:
이거는 제 개인적인 생각인데 만약 더 추상화한다면 주석이 이렇게 묶일게 아니라

```suggestion
            // 2. 구분자 처리 및 숫자 추출
            List<String> numberStrings = delimiterService.processInputAndExtractNumbers(input);
            List<BigDecimal> numbers = numberService.processNumbers(numberStrings);
            
            // 3. 숫자 계산
            BigDecimal result = numberService.calculateSum(numbers);
```

이게 좀 더 좋지않을까요 ? 그냥 코드스타일의 차이인것 같긴합니다 ㅎㅎ

Reply:
그렇네요.. 코드 수정하면서 주석을 알맞게 수정하지 못한 것, 확인해주셔서 감사합니다.